x/cellarfees/client/rest/query.go

```diff
@@ -0,0 +1,29 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/peggyjv/sommelier/v4/x/cellarfees/types"
+)
+
+// QueryAccountRequestHandlerFn is the query APY REST Handler.
+func QueryAPYRequestHandlerFn(storeName string, clientCtx client.Context) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		clientCtx, ok := rest.ParseQueryHeightOrReturnBadRequest(w, clientCtx, r)
+		if !ok {
+			return
+		}
+
+		route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryAPY)
```

Comment:
It looks like the route as reference elsewhere is `/cellarfees/apy`, is adding the `custom` prefix a requirement here? I assume that you were able to get the queries to return the correct data, but curious.

Reply:
I'm not totally sure of it's purpose, but it was included in the Cosmos SDK modules' rest implementations. My best guess is that it's an ABCI for application defined routes?